### PR TITLE
[full ci] Drop old hack from tether (and run gofmt -s)

### PR DESCRIPTION
**Pull request by caglar10ur**
/cc @hickeng

### CODE REVIEW

**Comment by dougm** [comment]
lgtm

File: lib/portlayer/storage/image_test.go [hunk 1]
```diff
@@ -51,8 +51,8 @@ func TestImageCopy(t *testing.T) {
 		ParentLink: parentURL,
 		Store:      storeURL,
 		Metadata: map[string][]byte{
-			"1": []byte{byte(1)},
-			"2": []byte{byte(2)},
+			"1": {byte(1)},
```
Comment (fdawg4l) [inline lib/portlayer/storage/image_test.go]: TIL.

File: pkg/vsphere/disk/util.go [hunk 2]
```diff
@@ -46,7 +46,7 @@ func waitForPath(ctx context.Context, path string) error {
 	go func() {
 		t := time.NewTicker(200 * time.Microsecond)
 		defer t.Stop()
-		for _ = range t.C {
+		for range t.C {
```
Comment (fdawg4l) [inline pkg/vsphere/disk/util.go]: While we're here, should we close the ticker?

**Comment by fdawg4l** [comment]
lgtm

**Comment by caglar10ur** [comment]
♻️

**Comment by caglar10ur** [comment]
♻️

**Pull request merged**